README.md

```diff
@@ -46,7 +46,8 @@ The list of items you’ll need is also included below:
 
 ## Setup and configuration
 
-Use of balenaDash requires that you allocate more memory to the GPU. This is achieved by adding (or editing the existing) the **Device configuration variable** `RESIN_HOST_CONFIG_gpu_mem`, for this project we recommend setting it to `396`.
+You can deploy this project to your device simply by clicking this button:
```

Comment:
I am thinking if we should drop the text completely and just have the button/image, @chrisys what do you think?

Reply:
I don't mind a bit of text as it's a new thing people haven't seen before so happy to be verbose around it until the flow is established. Perhaps word something like the below though?
```suggestion
This project can be deployed to a new balenaCloud application in one click using the button below:
```

Reply:
Fair enough, in that case let's at least make the button be in a new line so it is left-aligned.